certified-connectors/Connect2All/apiDefinition.swagger.json

```diff
@@ -779,22 +379,31 @@
             "$ref": "#/parameters/Company"
           },
           {
-            "$ref": "#/parameters/Configuration"
-          },
-          {
-            "$ref": "#/parameters/CsvDelimiter"
+            "$ref": "#/parameters/SystemId"
           },
           {
-            "$ref": "#/parameters/CsvHeaders"
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReadRequestSimpleSchema"
+            },
+            "x-ms-visibility": "internal"
           }
         ],
-        "produces": [
-          "text/plain"
-        ],
         "responses": {
           "200": {
             "description": "Success",
-            "schema": {}
+            "schema": {
+              "type": "object",
+              "description": "The schema for this read request",
```

Comment:
Hello @GAC-AJO, 

I have noticed that some places, it is readrequest and read request as well.  Please correct the same.

Reply:
Hello @vmanoharas,

I have done the requested changes. Please review again.